sdk/src/main/java/com/qonversion/android/sdk/QProductCenterManager.kt

```diff
@@ -326,6 +357,8 @@ class QProductCenterManager internal constructor(
 
             override fun onError(error: QonversionError) {
                 isLaunchingFinished = true
+                isProductsLoaded = true
```

Comment:
The **isLaunchingFinished** value is set to true only in 347 and 359 rows (always onSuccess and onError fun).
So we can get rid of this field by creating function that checks launchError and launchResult for nullability:
`private fun isLaunchingFinished() = launchError != null || launchResult != null`